<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DragGAN</class>
 <widget class="QMainWindow" name="DragGAN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="7,0,0,0,0" columnstretch="1,3">
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="2" column="0">
     <widget class="QWidget" name="experiment" native="true">
      <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0,0,0,0">
       <item row="6" column="3">
        <widget class="QCheckBox" name="SixBlock_CheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Block_ButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QCheckBox" name="SevenBlock_CheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Block_ButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="TargetImage_Label">
         <property name="text">
          <string>Target Image</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="FourBlock_CheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Block_ButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="TestTimes_Label">
         <property name="text">
          <string>Test times</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="FiveBlock_CheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Block_ButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="OnePoint_CheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>1 point</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Point_ButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Experiment_Label">
         <property name="text">
          <string>Experiment</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="Test_PushButton">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QCheckBox" name="SixtyEightPoints_CheckBox">
         <property name="text">
          <string>68 points</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Point_ButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="LatentBlock_Label">
         <property name="text">
          <string>Latent Block</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="FivePoints_CheckBox">
         <property name="text">
          <string>5 points</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Point_ButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="TestTimes_LineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="TargetImage_LineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QToolButton" name="TargetImage_ToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="SaveExperiment_PushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="DragTimes_Label">
         <property name="text">
          <string>Drag times</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="DragTimes_LineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>200</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="Experience_PushButton">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="model" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>14</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0,0,0,0,0,0,0" columnstretch="2,0,0,0,0,0,0">
       <item row="8" column="1">
        <widget class="QPushButton" name="Generate_PushButton">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="Seed_Label">
         <property name="text">
          <string>Seed</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="SaveReal_PushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="Wp_CheckBox">
         <property name="text">
          <string>W+</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">LatentSpace_ButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="RandomSeed_CheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Random Seed</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="W_CheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>W</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">LatentSpace_ButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Pickle_Label">
         <property name="text">
          <string>Pickle</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="Latent_Label">
         <property name="text">
          <string>Latent</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Device_Label">
         <property name="text">
          <string>Device</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="Plus4Seed_PushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="Minus4Seed_PushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="6">
        <widget class="QComboBox" name="Device_ComboBox"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="Seed_LineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Embedding_Label">
         <property name="text">
          <string>Embedding</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="Embedding_LineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="EmbeddingBrowse_PushButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="Reset4StepSize_PushButton_2">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="Pickle_LineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="Browse_PushButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="Recent_PushButton">
         <property name="text">
          <string>Recent...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="5">
     <widget class="ImageWidget" name="Image_Widget" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="drag" native="true">
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0,0,0,0">
       <item row="0" column="1" colspan="3">
        <widget class="QPushButton" name="AddPoint_PushButton">
         <property name="text">
          <string>Add point</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="Steps_Label">
         <property name="text">
          <string>Steps:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="R3_LineEdit"/>
       </item>
       <item row="9" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="R1_Label">
         <property name="text">
          <string>R1</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QPushButton" name="Reset4R2_PushButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="Stop_PushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="Reset4StepSize_PushButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="R1_LineEdit"/>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="R2_LineEdit"/>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="StepNumber_Label">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Drag_Label">
         <property name="text">
          <string>Drag</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="StepSize_Label">
         <property name="text">
          <string>Step Size</string>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QCheckBox" name="Optimize_CheckBox">
         <property name="text">
          <string>optimize</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QPushButton" name="Reset4R3_PushButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="Reset4R1_PushButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="StepSize_LineEdit"/>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QPushButton" name="Start_PushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLabel" name="R3_Label">
         <property name="text">
          <string>R3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="ResetPoint_PushButton">
         <property name="text">
          <string>Reset point</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="SaveGenerate_PushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="R2_Label">
         <property name="text">
          <string>R2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">components.ImageWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="LatentSpace_ButtonGroup"/>
  <buttongroup name="Block_ButtonGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="Point_ButtonGroup"/>
 </buttongroups>
</ui>
